backend/src/services/postType.service.js

```diff
@@ -51,3 +51,12 @@ exports.createPostType = async ({
     postyType: formatters.formatPreviewPostType(postyType),
   };
 };
+
+exports.getPostTypeDetail = async ({ communityId, postTypeId }) => {
+  const postType = await PostType.findById(postTypeId).lean();
+
```

Comment:
if (postType.community.toString()!=communityId){
    throw new ApiError(httpStatus.NOT_FOUND, 'Community ID does not match');
  }